### ISSUE TITLE
Support of the oemof foundation via institutes

### ISSUE THREAD

**Issue by busiing** [oemof/organisation#61]
As discussed in #44, we should ensure that the establishment of an oemof foundation and the foundation can be supported (>= 0) by the respective institutions.

Deadline: 04.05.2020

Please check in your institutions for a statement!
Is your Institute supporting (>=0) the oemof foundation?
- [ ] RLI
- [ ] Beuth
- [ ] Flensburg
- [ ] Oldenburg
- [ ] Bremen

Can your Institute be a member of a foundation? (Yes = checked box, No = unchecked box)
- [ ] RLI
- [ ] Beuth
- [ ] Flensburg
- [ ] Oldenburg
- [ ] Bremen

(Write missing institutions in the comments)

> > Es müssen 7 Mitglieder sein --> können das auch Institutionen sein?
> 
> By law, it is allowed to have not only natural persons but also legal entities as members. However, the foundations constitution may state something different. This might be advised, e.g. if institutes that want to contribute relevant amounts cannot be members because of their own rules. These might be reluctant to do so if other (possibly smaller) contributors got more visibility due to a formal membership.

**Comment by busiing** [comment]
@jnnr Could you ask for an opinion on this topic in the RLI?
@FranziPl @jakob-wo Could one of you ask for an opinion on this topic at the Beuth HS?
@p-snft Could you ask for an opinion on this topic in the DLR?

**Comment by c-moeller** [comment]
I checked the box for RLI.

**Comment by busiing** [comment]
I checked the box for OFFIS. We have the ability to support via membership or meeting fee.
Since there are just 2 ppl working with oemof at OFFIS a membership is not necessary for now.

**Comment by p-snft** [comment]
Please note that there is no such thing as "DLR Oldenburg". We are the Oldenburg site of (much bigger) DLR. Thus, memberships are a tough thing. However, other forms of collaboration might still be possible. In total, I would suggest a model that does not rely on formal memberships of institutions, but create a lean foundation that focuses on offering easy collaborations.

**Comment by joroeder** [comment]
> 
> 
> Please note that there is no such thing as "DLR Oldenburg". We are the Oldenburg site of (much bigger) DLR. Thus, memberships are a tough thing. However, other forms of collaboration might still be possible. In total, I would suggest a model that does not rely on formal memberships of institutions, but create a lean foundation that focuses on offering easy collaborations.

Yes, I agree. Actually, I am not sure whether having institutions as members is a good idea anyway ;) If the aim of the oemof foundation is to have an institutional independent organization, this option would be somehow counterproductive. However, the basis for collaboration is import though. I think a very important aspect is that the foundation relies on personal contacts and natural persons. If an institution becomes a member, this also dependents in many cases on single persons, e.g. a group-leader of an institute who's group is using oemof. If the group-leader switches the job, and he managed that the institute is a oemof member, and nobody works with oemof anymore, then why should the institute still be a member? Okay, this is a specific case, and every case will be different. I just want to underline, that the personal aspect is very important ;)

**Comment by busiing** [comment]
@p-snft mentioned that DLR Oldenburg wants to support the foundation.

**Comment by busiing** [comment]
After the creation of a [foundation](https://github.com/oemof/oemof/issues/71) is completed - let's start to gather supportive institutional members to pay our bills :)

- [] RLI
- [] Flensburg
- [] Oldenburg (DLR)
- [ ] Oldenburg (OFFIS)
- [] Bremen
